### ISSUE TITLE
Multiple tooltips on single page on mobile are not working

### ISSUE THREAD

**Issue by Filipeue** [bsidelinger912/react-tooltip-lite#84]
When more than 1 tooltip is placed on single page and viewed ON MOBILE (you cannot reproduce this bug in chrome mobile emulator - you need real device) only first you interact with is working properly. Then when other tooltips are clicked, they are not opened.
Is some special key / id required for it to work properly?

**Comment by bsidelinger912** [comment]
Hey @Filipeue, I'll try and take a look at this, have you tested with both ios and android? which browsers?

**Comment by Filipeue** [comment]
I have tested it on OnePlus 7T Pro, Adnroid 10, Chrome 81.0.4044.138. Unfortunately the issue seems not to be present when using tool like Browserstack (https://www.browserstack.com/), no matter the browser or device I choose. So I tested it on my other device Android 5.0.1, same chrome version and the issue persited. Then iphone 10s Safari: issue NOT present.
So it seems to be android + chrome issue and to simulate you need real device preferably with newest Chrome.

**Comment by bsidelinger912** [comment]
Okay thanks @Filipeue, and by "interact" I assume you mean tapping the target?

**Comment by Filipeue** [comment]
yeah I meant tapping the target, but I have tested it on few more android devices with newest chrome and I cannot simulate the issue, so I am starting to think It has something to do with my chrome profile or something... really weird...
on 2 of my own devices I see the issue and everywhere else it is working just fine... after this finding the issue is not as important as it seemed, so if you choose to close it I can understand
